src/_sass/_site.scss

```diff
@@ -370,32 +383,32 @@ main .content {
 }
 
 .card-grid {
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  justify-content: flex-start;
-  margin-left: -15px;
-  margin-right: -15px;
+  display: grid;
+  grid-auto-rows: minmax(5rem, auto);
+  gap: 1rem;
+  margin-top: 1rem;
+}
+.card-col2 {
+  grid-template-columns: repeat(2, 1fr);
+}
+.card-col3 {
+  grid-template-columns: repeat(3, 1fr);
 }
 
 .card {
   display: block;
-  position: relative;
-  width: 300px;
-  height: auto;
-  min-height: 100px;
-  padding: 20px;
-  margin: 15px;
+  padding: 1rem;
   overflow: hidden;
-  box-sizing: border-box;
-  background-color: #294058;
-  border-radius: 0;
+  background-color: $gray-lighter;
+  border: 2px solid $flutter-color-dark-blue;
```

Comment:
I don't like this new style, I prefer the old one. We worked with a UI designer to get this right.

Reply:
I've worked with the design team on these things too. My concern is that with so many cards, the page might become overwhelmingly dark. I'm happy to change it, but that's the reasoning.